orttraining/orttraining/test/python/orttraining_test_dort.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import unittest
+
+import torch
+import torchdynamo
+from torch import nn
+from torch.nn import functional as F
+
+from onnxruntime.training.torchdynamo.register_backend import aot_ort
+
+# Make computation deterministic.
+torch.manual_seed(42)
+
+
+class TestTorchDynamoOrt(unittest.TestCase):
+    """Containers of tests for TorchDynamo ORT backend."""
+
+    def test_elementwise_model(self):
+        """Test DORT with a pure function."""
```

Comment:
```suggestion
        """Test ORT with a pure function."""
```
typo or DORT stands for Dynamo ORT?

Reply:
filename has dort too

Reply:
DORT=Dynamo ORT. lol. I will change it to `DORT (Dynamo-ORT)` inside Python file.